test/unit/syscalls-specs.js

```diff
@@ -107,6 +107,21 @@ describe('System calls', withMocks({teen_process}, (mocks) => {
 }));
 
 describe('System calls',  withMocks({adb, B, teen_process}, (mocks) => {
+  it('should return adb version', async () => {
+    mocks.adb.expects("adbExec")
+      .once()
+      .withExactArgs('version')
+      .returns("Android Debug Bridge version 1.0.39\nRevision 5943271ace17-android");
+    let adbVersion = await adb.getAdbVersion();
+    adbVersion.versionString.should.equal("1.0.39");
+    adbVersion.versionFloat.should.be.within(1.0, 1.0);
+    adbVersion.major.should.equal(1);
+    adbVersion.minor.should.equal(0);
+    adbVersion.patch.should.equal(39);
+    // verify adb exec is called once, since we are caching the result
```

Comment:
Oh! Good point.

https://lodash.com/docs/4.17.4#memoize
"The cache is exposed as the cache property on the memoized function. Its creation may be customized by replacing the _.memoize.Cache constructor with one whose instances implement the Map method interface of clear, delete, get, has, and set."